waterlog/src/Pages/Home.js

```diff
@@ -10,7 +10,7 @@ class HomeComponent extends Component {
         <AlertsMap />
         <Wastage />
         <p className="home-text">Tank Levels</p>
-        <TankLevels />
+        {/*<TankLevels />*/}
```

Comment:
gc

Reply:
Also, if you remove <TankLevels/> be sure to remove its import if not used anymore
`import TankLevels from './../Components/TankLevels';`

Reply:
The code has errors, We are still to fix them

Reply:
please add a TODO with description

Reply:
I will remove the comment since it does not affect my feature